test/test_api/test_api.py

```diff
@@ -315,9 +417,7 @@ def test_tabular_regression(openml_name, resampling_strategy, backend):
         model = estimator._backend.load_cv_model_by_seed_and_id_and_budget(
             estimator.seed, successful_num_run, run_key.budget)
         assert isinstance(model, VotingRegressor)
-        assert len(model.estimators_) == 5
-        assert isinstance(model.estimators_[0].named_steps['network'].get_network(),
-                          torch.nn.Module)
+        assert len(model.estimators_) == 2
```

Comment:
This is `num_splits`, right?
Can we avoid hard-coding?

Reply:
yes, makes sense. I'll change this